.gitignore

```diff
@@ -12,6 +12,7 @@
 # production
 /build
 /dist
+/dest
```

Comment:
The name is a bit off, maybe `stories` suits this one better?

Reply:
Agreed, I'll change it to something else. It's not `stories` though, the stories are under `dest/demo`.